doc/source/users_guide/syc_datamodel.rst

```diff
@@ -98,7 +98,7 @@ data model, as shown in the following code examples.
 You can access unnamed objects, such as ``analysis_control`` attributes, even when
 they are initially *empty*. You can confirm this using the ``print_state()`` method:
 
-.. code-block:: python
+.. code-block:: none
```

Comment:
These code blocks don't, strictly speaking, contain Python and `blacken-docs` does not like them. In one case, it was falling over with an exception and I had to do some work to find out which file was causing it.